python/equistore-operations/setup.py

```diff
@@ -0,0 +1,120 @@
+import os
```

Comment:
Cant't we use code from the setup.py of equistore-core? Since operations should depend on the core it should be there.

Reply:
done!